components/dashboard/src/components/UsageBasedBillingConfig.tsx

```diff
@@ -47,6 +178,11 @@ export default function UsageBasedBillingConfig({
         <div className="mb-16">
             <h2 className="text-gray-500">Manage usage-based billing, usage limit, and payment method.</h2>
             <div className="max-w-xl flex flex-col">
+                {billingError && (
+                    <Alert className="max-w-xl mb-4" closable={false} showIcon={true} type="error">
```

Comment:
Good idea moving this alert here, so that it's encapsulated in the common component. 👍

However, the margins need to be adjusted:

| BEFORE | AFTER |
| --- | --- |
| <img width="629" alt="Screenshot 2022-09-15 at 09 21 26" src="https://user-images.githubusercontent.com/599268/190340750-1eb0cdc8-ffd1-4926-92c4-d88d9ddbaf0d.png"> | <img width="620" alt="Screenshot 2022-09-15 at 09 23 14" src="https://user-images.githubusercontent.com/599268/190341018-9a3574bf-c328-4400-86b2-18822a8e848b.png"> |

```suggestion
                    <Alert className="max-w-xl mt-2" closable={false} showIcon={true} type="error">
```